packages/expo/src/Notifications.ts

```diff
@@ -46,6 +46,9 @@ type Channel = {
 // Since that's not supported on iOS, we generate an unique string.
 type LocalNotificationId = string | number;
 
+const RED = 2;
```

Comment:
Use TypeScript enums instead, using a plural name because the enum values are bit masks:

```
enum NotificationCategoryFlags {
  None = 0,
  AuthenticationRequired = 1,
  Destructive = 2,
  Foreground = 4,
};
```

The values of these options do not need to correspond to the runtime values that iOS uses because Android may offer different options and likely doesn't use the same enum values. There should be an Expo-to-iOS conversion step. 

Have you looked at the options Android provides?